templates/agent.docker.yaml

```diff
@@ -62,23 +68,18 @@ identities:
     dst_port: 443
   pushdown: []
 - node:
-    name: googleapis_storage
-    dns_pattern: .*storage.googleapis.com
-    dst_port: 443
-  pushdown: []
-- node:
-    name: gcr_io
-    dns_pattern: gcr.io
+    name: ghcr
+    dns_pattern: ghcr.io
     dst_port: 443
   pushdown: []
 - node:
-    name: gcr_io_asia
-    dns_pattern: asia.gcr.io
+    name: googleapis_storage
+    dns_pattern: .*storage.googleapis.com
     dst_port: 443
   pushdown: []
```

Comment:
dns_pattern should be in pushdown - whenever there is a regex and you expect it to expand to more than a handful. Otherwise policies blow up